.prettierignore

```diff
@@ -6,3 +6,4 @@ docs/
 resources/
 sampleFlows/
 coverage/
+**/docs/**/*.md
```

Comment:
I'm excluding docs for now as they're automatically generated and currently they error on prettier.